YellowNotes.Api/Startup.cs

```diff
@@ -22,6 +24,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.AddDbContextPool<DatabaseContext>(options => 
                 options.UseMySql(Configuration.GetConnectionString("Development")));
+            services.AddSingleton<IEmailConfiguration>(Configuration.GetSection("EmailConfiguration").Get<EmailConfiguration>());
```

Comment:
Typically we configure in such way configurations: `services.Configure<SomeConfiguration>(_configuration.GetSection("SomeSection"));`